cerebras_tools_proxy.py

```
@@ -0,0 +1,562 @@
+                    
+                    # Enhance response with tool results
+                    enhanced_content = content + tool_results
+                    anthropic_response["content"][0]["text"] = enhanced_content
+                    print(f"✅ Tools executed: {len(tool_requests)} operations")
+            
+            print(f"✅ Response ready ({len(content)} chars)")
+            return JSONResponse(anthropic_response)
+            
+        except requests.exceptions.RequestException as e:
+            print(f"❌ Cerebras request failed: {e}")
+            raise HTTPException(status_code=503, detail=f"Cerebras service unavailable: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)